src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/cimChildJobBase.cs

```diff
@@ -1087,6 +1087,7 @@ protected override void Dispose(bool disposing)
                 }
 
                 _cimSensitiveValueConverter.Dispose();
+                _cancellationTokenSource.Dispose();
```

Comment:
This comes from Windows PowerShell.
@rjmholt your team could internally consider a backport.